docs/layers/geojson-layer.md

```diff
@@ -7,70 +7,33 @@
 The GeoJson Layer takes in [GeoJson](http://geojson.org/) formatted data and
 renders it as interactive polygons, lines and points.
 
-  <div align="center">
-    <img height="300" src="/demo/src/static/images/demo-thumb-geojson.jpg" />
-  </div>
-
-```
-import {GeoJsonLayer} from 'deck.gl';
-
-new GeoJsonLayer({
-  data: {
-    "type": "FeatureCollection",
-      "features": [
-        {
-          "type": "Feature",
-          "geometry": {
-            "type": "Point",
-            "coordinates": [102.0, 0.5]
-          },
-          "properties": {
-            "color": [255, 0, 0, 255],
-            "prop1": {"application": "defined"}
-          }
-        },
-        ...
-      ]
-    }
-  }
-});
-```
-
-Remarks:
-* By supplying a GeoJson `FeatureCollection` you can render multiple polygons,
-  lines and points.
-* Each Feature has an optional "properties" object. The layer will look
-  for an optional property `color`, which is expected to be a 4 element
-  array of values between 0 and 255, representing the rgba values for
-  the color of that `Feature`.
+<div align="center">
+  <img height="300" src="/demo/src/static/images/demo-thumb-geojson.jpg" />
+</div>
 
+    import {GeoJsonLayer} from 'deck.gl';
 
 ## Properties
 
 Inherits from all [Base Layer properties](/docs/layers/base-layer.md),
 however, the `data` prop is interpreted slightly more flexibly to accommodate
-puree GeoJson "payloads".
+pure GeoJson "payloads".
 
-Also note that the `GeoJsonLayer` is a **composite** layer, that renders points
-using `ScatterplotLayer`, lines and polygon outlines (`stroked` polygons)
-using `PathLayer`, and `filled` polygons using a primitive polygon layer.
-The `GeoJsonLayer` passes through most of its props to these underlying layers
-so you can sometimes achieve additional control of rendering by studying
-the props of the generated layers.
+Also note that the `GeoJsonLayer` is a **composite** layer, that renders points, lines and polygons with its outlines with different "underlying" layers, so developers can sometimes achieve additional control of rendering by controlling
```

Comment:
Just a thought: If we don't document what the generated layers are, I think we need to make an effort to declare all the props from those layers in this layer.

E.g. how is a user to know that he can use `radiusScale` to control the radius of the points, etc, if it is not mentioned here, and the fact that we generate a `ScatterplotLayer` isn't mentioned either.

At the very least, I think we need to list the fp64 prop here, as we pass it down to all the sublayers.